### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by adayton1**
* Update BLT to v0.6.1
* Update CAMP to v2024.02.0
* Update Umpire to v2024.02.0
* Update RAJA to v2024.02.0
* Update CHAI to v2024.02.0
* Update Spack to develop-2024-01-21
* Update radiuss-spack-configs to [c585417](https://github.com/LLNL/radiuss-spack-configs/commit/c5854178b618b882386dd5f3c82fd34528b85522)
* Use local spack package for CARE and then radiuss-spack-configs spack packages for dependencies
* Fix up CARE install
* Use new way of exporting BLT logic
* Update local CARE spack package

### CODE REVIEW

**Review by dtaller — APPROVED** [review]

File: scripts/spack_packages/packages/care/package.py [hunk 1]
```diff
@@ -37,22 +37,31 @@ class Care(CachedCMakePackage, CudaPackage, ROCmPackage):
     variant('implicit_conversions', default=False, description='Enable implicit conversions to/from raw pointers')
     variant('loop_fuser', default=False, description='Enable loop fusion capability')
 
-    depends_on('cmake@3.14.5:', type='build')
-    depends_on('cmake@3.21:', when='+rocm', type='build')
+    depends_on('cmake@3.18:', type='build', when='@develop')
+    depends_on('cmake@3.21:', type='build', when='+rocm')
+    depends_on('cmake@3.14:', type='build', when='@0.10.0:')
```
Comment (agcapps) [inline scripts/spack_packages/packages/care/package.py:42]: Maybe a small thing, but if possible, please order in descending version order like the BLT lines below.  This just helps with my understanding.

**Review by agcapps — APPROVED** [review]
Thank you, @adayton1 .

File: scripts/spack_packages/packages/care/package.py (hunk 1 above)
Comment (adayton1) [reply scripts/spack_packages/packages/care/package.py]: Fixed.

**Comment by dtaller** [comment]
Hi @adayton1 . In your last MR, https://github.com/LLNL/CARE/pull/245, adriene bernede suggested a few changes, but it was too late to include them since you already merged. 

Maybe fix the license mistake that adriene found there (files that mention the CHAI license instead of the care one due to a copy/paste issue)?

**Comment by adayton1** [comment]
> Hi @adayton1 . In your last MR, #245, adriene bernede suggested a few changes, but it was too late to include them since you already merged.
> 
> Maybe fix the license mistake that adriene found there (files that mention the CHAI license instead of the care one due to a copy/paste issue)?

I'm going to take a pass at updating all the copyright dates in a follow on merge request, and I'll fix the license mistake in that branch.

**Comment by adrienbernede** [comment]
@adayton1 I think we should consider placing your custom CARE package in the RADIUSS Spack Configs repo. That’s where we centralize the local spack packages. Would you be fine with that ?
Not a high priority item, but I’d like your agreement for when the time comes.

**Pull request merged**